Add 2 labs to non-free-tier version of advanced ADW workshop "Analyzing Your Data with Autonomous Database", and fixed the Create Schema lab in "Get Started with Oracle Database" workshop

greenrick opened the pull request:
I'm adding 2 labs to the **non-free-tier** versions of the ADW "advanced" workshop named "Analyzing Your Data with Autonomous Database":
  - Creating Refreshable Clones in Autonomous Database
  - Setting Event and Alarm Notifications for an Autonomous Database
I added them to 3 of the 5 manifest.json files that I find in the GitHub learning-library repository, before the "Getting Started with Machine Learning Notebooks" lab. Since FREETIER doesn't support refreshable clones (nor disaster recovery via Autonomous Data Guard), I didn't update the 2 manifest.json files for freetier. Here are the 3 manifest.json files I updated:
shared\adb-advanced-workshop\livelabs\manifest.json
shared\workshops\livelabs-indepth\manifest.json
shared\workshops\livelabs2-indepth\manifest.json

Also fixed the Create Schema lab in Get Started with Oracle Database workshop:
learning-library\data-management-library\database\db-quickstart\db-create-schema\db-create-schema.md
Instead of Revoke CONNECT, Revoke CREATE SESSION

tmcginn reviewed (changes requested):
Rick, before I approve, please test your new labs using the existing green button setup. We need to make sure that the new labs don't require additional policies or changes to the setup to work.

tmcginn commented:
@greenrick - can you setup up time today on my cal so we can resolve this?

greenrick commented:
Added NOTE to "Creating Refreshable Clones in Autonomous Database" and "Setting Event and Alarm Notifications for an Autonomous Database" labs, stating these features aren't available in Free Tier Autonomous Databases. You can run the lab by reserving a hosted Livelabs environment, or by signing up for free trial credits, or by using your own tenancy.

Temporarily removed those 2 labs from the LiveLabs manifest.json files, so they won't appear in the navigation tree until LiveLabs team changes the LiveLabs environment policies to allow Rick Green to test them in a LiveLabs reserved session.

Also fixed the Create Schema lab in Get Started with Oracle Database workshop:
learning-library\data-management-library\database\db-quickstart\db-create-schema\db-create-schema.md
Instead of Revoke CONNECT, Revoke CREATE SESSION

Also added a line space to ADW Disaster Recovery lab, to separate 2 NOTEs
